scene/gui/popup_menu.cpp

```diff
@@ -79,7 +79,7 @@ Size2 PopupMenu::get_minimum_size() const {
 			size.width += check_w + hseparation;
 		}
 
-		String text = items[i].shortcut.is_valid() ? String(tr(items[i].shortcut->get_name())) : items[i].xl_text;
+		String text = items[i].shortcut.is_valid() ? items[i].xl_text + String(tr(items[i].shortcut->get_name())) : items[i].xl_text;
```

Comment:
Seems fine, though you can maybe write it as:
```
String text = items[i].xl_text;
if (items[i].shortcut.is_valid()) {
	text += String(tr(items[i].shortcut->get_name()));
}
```

Reply:
Also, how does it look like after this fix? Isn't there a need for a space between the `xl_text` and the shortcut name?

Reply:
> 
> 
> Seems fine, though you can maybe write it as:
> 
> ```c++
> String text = items[i].xl_text;
> if (items[i].shortcut.is_valid()) {
> 	text += String(tr(items[i].shortcut->get_name()));
> }
> ```

On further inspection `tr(items[i].shortcut->get_name())` is not needed, it always returns an empty string, with or without shortcut.

The actual drawing of the shortcut is done [here](https://github.com/godotengine/godot/blob/1794b95fd2812b8fdff99258b4a6dc25818b58c4/scene/gui/popup_menu.cpp#L520)

> 
> 
> Also, how does it look like after this fix? Isn't there a need for a space between the `xl_text` and the shortcut name?

Here's how it looks:

![image](https://user-images.githubusercontent.com/19125153/52720693-65c60c00-2fa8-11e9-964a-04ce8c1704d5.png)

Reply:
Nice! So I guess it should be amended to simply `String text = items[i].xl_text;`?

Reply:
Yes, exactly